### ISSUE TITLE
[BUG] Menu doesn't go away after pressing escape

### ISSUE THREAD

**Issue by ParaliyzedEvo** [stoneman2/LethalProgression#40]
**Mod Version**
v1.3.2

**Describe the bug**
Pretty much when I go to skills, my points are at 99999 and it should be 18. Also when I press escape the menu doesn't go away

**To Reproduce**
1. Press escape
2. Go to skills
3. See that you have 99999 points which is not normal
4. Press escape to exit the menu
5. Menu still shows up even though I'm not in the settings menu

**Expected behavior**
When I press escape the menu and the skill points should be 18 and not 99999

**Screenshots**
![image](https://github.com/stoneman2/LethalProgression/assets/107338855/f16fadb3-1744-44e5-a884-4c4e0142823b)

**Mod list:**
Lethal Progression
atg-FreeJester-1.0.3
itsmeowdev-DoorFix-1.0.0
boring-InfiniteShotgunAmmo-1.0.3
NutNutty-AlwaysPickup-1.0.0
Backrooms-Backrooms-0.0.6
TeamClark-BigBossSuit-1.0.1
no00ob-LCSoundTool-1.4.0
Clementinise-CustomSounds-2.1.2
HoffmanTV-angry_british_bug-1.3.0
Crab_imitation-Space_Pajamas_Suit-1.0.6
ViViKo-MoreMaterials-1.3.0
Monkeytype-HideChat-1.0.0
FlipMods-ObjectVolumeController-1.0.3
sunnobunno-BonkHitSFX-1.0.4
togglemybits-Strange_Objects-1.2.1
MassiveNewCoilers-FixCentipedeLag-2023.12.7
FlipMods-FasterItemDropship-1.2.0
steven4547466-YoutubeBoombox-1.5.0
Blorb-WeatherMultipliers-1.0.0
FlipMods-ReservedItemSlotCore-1.4.4
FlipMods-ReservedWalkieSlot-1.4.6
FlipMods-ReservedFlashlightSlot-1.4.5
Renegades-WalkieUse-1.3.1
TeamClark-CombineEliteSuit-1.0.2
PanHouse-LethalClunk-1.1.1
299792458-MoreMoneyStart-2.1.0
yeahimliam-Peter_Griffin_Suit-1.0.1
BlueAmulet-LCBetterClock-1.0.3
raydenoir-SCP173CoilheadSFX-1.2.3
Isbjorn52-MetalPipe-1.0.0
Ozone-Disregard_Early_Voting-1.0.2
TeamClark-ODST_Suit-1.0.0
BepInEx-BepInEx_MLLoader-2.1.0
alexanderjoe-LC_Symphony-1.2.0
alexanderjoe-BiggeryBattery-1.0.2
matsuura-HealthMetrics-1.0.2
taffyko-NameplateTweaks-1.0.5
kRYstall9-FastSwitchPlayerViewInRadar-1.3.1
quackandcheese-MirrorDecor-1.1.6
monkes_mods-JumpDelayPatch-1.0.1
Radsi2-Shotgun-1.0.0
HolographicWings-LethalExpansion-1.2.16
EliteMasterEric-WackyCosmetics-1.1.0
EladNLG-EladsHUD-1.1.0
SirTyler-BetterTeleporter-1.2.1
anormaltwig-TerminalExtras-1.0.1
TeamClark-SCP_Foundation_Suit-1.0.1
HomelessGinger-MaskedEnemyOverhaul-2.1.0
MoreLethal-Pinger-3.2.0
Rune580-LethalCompany_InputUtils-0.4.2
Thorlar-HealthStation-1.0.1
PopleZoo-BetterItemScan-2.1.6
Drakorle-MoreItems-1.0.1
malco-Lategame_Upgrades-2.7.0
Tyzeron-Minimap-1.0.5
FlipMods-BetterStamina-1.3.0
Suskitech-AlwaysHearActiveWalkies-1.4.3
Renegades-FlashlightToggle-1.4.1
Evaisa-HookGenPatcher-0.0.5
BepInEx-BepInExPack-5.4.2100
AllToasters-QuickRestart-1.2.0
NotAtomicBomb-Terminal_Clock-1.0.5
xCeezy-LethalEscape-0.7.6
amnsoft-EmployeeAssignments-1.0.8
femboytv-LethalPaintings-1.2.0
Ozone-BepInUtils-1.2.1
5Bit-VoiceHUD-1.0.4
Hexnet111-SuitSaver-1.1.2
Midge-PushCompany-1.2.0
Pooble-LCBetterSaves-1.5.0
ATK-ShipClock-0.9.0
femboytv-LethalPosters-1.2.0
AllToasters-SpectateEnemies-1.5.0
OrtonLongGaming-FreddyBracken-1.0.6
Evaisa-LethalThings-0.7.1
tinyhoot-ShipLoot-1.0.0
RickArg-Helmet_Cameras-2.1.5
FlipMods-MoreBlood-1.0.2
Mom_Llama-Lethal_Company_Enhancer-0.1.8
Rozebud-FOV_Adjust-1.1.1
2018-Brutal_Company-1.2.4
vasanex-ItemQuickSwitch-1.1.0
NotAtomicBomb-TerminalApi-1.4.0
FlipMods-LetMeLookDown-1.0.1
Sligili-HDLethalCompany-1.5.6
x753-Mimics-2.2.1
Evaisa-LethalLib-0.7.0
RugbugRedfern-Skinwalkers-2.0.1
Sligili-More_Emotes-1.2.2
anormaltwig-LateCompany-1.0.6
x753-More_Suits-1.4.1
notnotnotswipez-MoreCompany-1.7.2
2018-LC_API-2.2.0
Stoneman-LethalProgression-1.3.2
Paraliyzed-Pick_Up_the_Sheriff_for_Me_Baby-1.0.1

**Additional context**
No comment

**Comment by stoneman2** [comment]
There's so many mods, can you remove some and see what is incompatible? Does this happen with Lethal Progression alone?

**Comment by ParaliyzedEvo** [comment]
Oh sorry about that, pretty much the mod you just need to look at is 'Paraliyzed-Pick_Up_the_Sheriff_for_Me_Baby-1.0.1' as its a modpack with all the mods listed here. Also I tried the mod alone and sometimes it happens sometimes it doesn't which is weird.

**Comment by stoneman2** [comment]
Please send log

**Comment by stoneman2** [comment]
You seemed to have installed some mods wrong, this causes a lot of problems for every mod

**Comment by ParaliyzedEvo** [comment]
Wdym? I installed every mod by using the modpack i metioned before from the thunderstore app like everyone installs mods unless ofc they want to install it manually.

**Comment by stoneman2** [comment]
The console is being spammed with the minimap mod, is that working correctly?

**Comment by ParaliyzedEvo** [comment]
In game, yes it shows up and works correctly when the bug I mentioned before doesn't happen.

**Comment by stoneman2** [comment]
You sent me the log on the one that worked? That's not very useful

**Comment by ParaliyzedEvo** [comment]
No, I sent you the one where the bug shows up. Besides I dont have any other logs anyways so I would've just sent you the one I sent.

**Comment by stoneman2** [comment]
That's strange, most likely some mod compatibility issue, I don't know if I can fix this without knowing which one, no time

**Comment by ParaliyzedEvo** [comment]
Yea, I agree on your opinion. I'll try and figure out what mod is causing it and I'll tell to tomorrow if you don't mind.

**Comment by ParaliyzedEvo** [comment]
Ok, so I tested it out. The only thing I have disabled is Brutal Company and Pick Up the Sheriff for Me Baby. I doubt that is the modpack but maybe it could be Brutal Company as it's the only one I have disabled since I don't play it normally. So maybe it's Brutal Company. Maybe you could test out somethings on your end since idk how to debug stuff.
